list/list.go

```diff
@@ -197,6 +197,42 @@ func Reduce[T collections.CollectionElement](l *List[T], callback listReduceFunc
 	return result
 }
 
+// Define a generic Sort method for the List structure.
```

Comment:
Can you make this comment more descriptive ? 
Ref: https://github.com/gopher-utils/go-collections/blob/1d35847621b6eb1e6ef8ba38c9217ad7aa56ab5c/list/list.go#L98

Reply:
alright, working on that